src/index.js

```diff
@@ -1,90 +1,97 @@
-import 'dotenv/config';
+import 'dotenv/config'
 
-import cors from 'cors';
-import express from 'express';
-import requestId from 'express-request-id';
-import rateLimit from 'express-rate-limit';
-import swaggerUi from 'swagger-ui-express';
-import helmet from 'helmet';
-import morgan from 'morgan';
-import nunjucks from 'nunjucks';
+import cors from 'cors'
+import express from 'express'
+import helmet from 'helmet'
+import morgan from 'morgan'
+import nunjucks from 'nunjucks'
+import requestId from 'express-request-id'
+import rateLimit from 'express-rate-limit'
+import routes from './routes'
+import serveIndex from 'serve-index'
+import swaggerDocument from '../swagger.json'
+import swaggerUi from 'swagger-ui-express'
+import utils from './utils'
+import models, { sequelize } from './models'
 
-import swaggerDocument from '../swagger.json';
-import models, { sequelize } from './models';
-import routes from './routes';
-import utils from './utils';
-
-const app = express();
+const app = express()
 const swaggerOptions = {
 	customCss: '.swagger-ui .topbar { display: none }'
-};
+}
 const apiLimiter = rateLimit({
 	windowMs: 15 * 60 * 1000, // 15 minutes
-	max: 100
-});
+	max: 1000
```

Comment:
What was the motivation for increasing this?

Reply:
Also a test with the mkdocs. With it at only 100 and serving mkdocs it hit a limit with just a few page loads.